support-services/src/test/scala/com/gu/support/catalog/CatalogServiceSpec.scala

```diff
@@ -26,15 +26,15 @@ class CatalogServiceSpec extends AsyncFlatSpec with Matchers {
       Monthly,
       HomeDelivery,
       Everyday
-    ) shouldBe Some(Price(62.79, GBP))
+    ) shouldBe Some(Price(67.99, GBP))
```

Comment:
Are these just because we haven't updated the test resources in a while?

Reply:
Yes exactly